pom.xml

```diff
@@ -134,7 +139,7 @@
           <target>1.7</target>
         </configuration>
       </plugin>
-      <plugin>
+      <!--<plugin>
```

Comment:
Why are you commenting this? This plugin should check the existance of license in every source file.